subcommands/keys/tuf_updates_apply.go

```diff
@@ -0,0 +1,58 @@
+package keys
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	applyCmd := &cobra.Command{
+		Use:   "apply",
+		Short: "Apply staged TUF root updates for the Factory",
+		Run:   doTufUpdatesApply,
+	}
+	applyCmd.Flags().StringP("txid", "x", "", "TUF root updates transaction ID.")
+	_ = applyCmd.MarkFlagRequired("txid")
+	tufUpdatesCmd.AddCommand(applyCmd)
+}
+
+func doTufUpdatesApply(cmd *cobra.Command, args []string) {
+	factory := viper.GetString("factory")
+	txid, _ := cmd.Flags().GetString("txid")
+
+	err := api.TufRootUpdatesApply(factory, txid)
+	if err != nil {
+		msg := "Failed to apply staged TUF root updates:\n%w\n"
+		var isNonFatal bool
+		if herr := client.AsHttpError(err); herr != nil {
+			nonFatalCodes := []int{400, 401, 403, 422, 423}
+			for _, code := range nonFatalCodes {
+				if code == herr.Response.StatusCode {
+					isNonFatal = true
+					break
+				}
+			}
```

Comment:
if we are on Go 1.18 or higher, then we might use:

```go
isNonFatal = slices.Contains([]int{400, 401, 403, 422, 423}, herr.Response.StatusCode)
```

Reply:
Thanks! I was looking for a neat one-liner like this.

Reply:
https://github.com/foundriesio/fioctl/pull/193/commits/c842e4e54801c2480d4708d48cc0d43018b65beb